charts/pravega-operator/README.md

```diff
@@ -4,16 +4,17 @@ Installs [pravega-operator](https://github.com/pravega/pravega-operator) to crea
 
 ## Introduction
 
-This chart bootstraps a [pravega-operator](https://github.com/pravega/pravega-operator) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager. The chart can be installed multiple times to create Pravega Operator on multiple namespaces.
+This chart bootstraps a [pravega-operator](https://github.com/pravega/pravega-operator) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
 
 ## Prerequisites
   - Kubernetes 1.15+ with Beta APIs
   - Helm 3.2.1+
-  - An existing Apache Zookeeper 3.6.1 cluster. This can be easily deployed using our [Zookeeper Operator](https://github.com/pravega/zookeeper-operator)
-  - An existing Apache Bookkeeper 4.9.2 cluster. This can be easily deployed using our [BookKeeper Operator](https://github.com/pravega/bookkeeper-operator)
   - Cert-Manager v0.15.0+ or some other certificate management solution in order to manage the webhook service certificates. This can be easily deployed by referring to [this](https://cert-manager.io/docs/installation/kubernetes/)
   - An Issuer and a Certificate (either self-signed or CA signed) in the same namespace that the Pravega Operator will be installed (refer to [this](https://github.com/pravega/pravega-operator/blob/master/deploy/certificate.yaml) manifest to create a self-signed certificate in the default namespace)
 
+## Deploying in Test Mode
```

Comment:
This section can be removed from here and from all user documentation, cause we almost never want users to run the operator in "test mode" except when a user needs to run "Hello World" workload on Pravega using minikube..... 
Instead this section can be part of the **Setting up of Pravega on minikube** document and part of the [development](https://github.com/pravega/pravega-operator/blob/master/doc/development.md) documentation for operator.
The "test mode" is more a feature for developers than for Pravega users.

Reply:
Removed